src/main/kotlin/telegram/Handling.kt

```diff
@@ -666,81 +646,96 @@ object MafiaHandler {
             parametrized(updateCheckCommand) {
                 updateCheck(str(0))
                 showAdmin(chatId, long(1), bot)
             }
             parametrized(hostRequestCommand) {
-                showHostRequests(long(0), chatId, bot)
+                showHostRequests(long(0), chatId, bot, int(1))
             }
             parametrized(hostSettingsCommand) {
-                showHostSettings(long(0), chatId, bot)
+                showHostSettings(chatId, long(0), bot, int(1))
             }
             parametrized(adminSettingsCommand) {
-                val messageId = long(0)
-                showAdminListMenu(chatId, messageId, bot)
+                showAdminListMenu(chatId, long(0), bot, int(1))
             }
             parametrized(timeLimitOnCommand) {
-                val res = bot.sendMessage(ChatId.fromId(chatId), "Введите срок действия разрешения в днях:")
-                if (res.isSuccess) {
-                    val desc = res.get().messageId
-                    createAdminContext(desc, AdminState.HOST_TIME)
-                }
+                val msgId = sendMessage(
+                    bot,
+                    chatId,
+                    "Введите срок действия разрешения в днях:"
+                )
+                updateMessage(
+                    bot, chatId, msgId,
+                    inlineKeyboard {
+                        createAdminContext(msgId, AdminState.HOST_TIME)
+                    }
+                )
             }
             parametrized(timeLimitOffCommand) {
                 hostInfos.update(long(0)) { timeLimit = false }
-                showHostSettings(long(1), chatId, bot)
+                showHostSettings(chatId, long(1), bot, long(0))
             }
             parametrized(gameLimitOnCommand) {
-                val res = bot.sendMessage(ChatId.fromId(chatId), "Введите количество разрешенных игр:")
-                if (res.isSuccess) {
-                    val desc = res.get().messageId
-                    createAdminContext(desc, AdminState.HOST_GAMES)
-                }
+                val msgId = sendMessage(
+                    bot,
+                    chatId,
+                    "Введите количество разрешенных игр:"
+                )
+                updateMessage(
+                    bot, chatId, msgId,
+                    inlineKeyboard { createAdminContext(msgId, AdminState.HOST_GAMES) }
+                )
             }
             parametrized(gameLimitOffCommand) {
                 hostInfos.update(long(0)) { gameLimit = false }
-                showHostSettings(long(1), chatId, bot)
+                showHostSettings(chatId, long(1), bot, long(0))
             }
             parametrized(shareCommand) {
                 hostInfos.get(long(0))?.let {
                     hostInfos.update(long(0)) { canShare = !it.canShare }
-                    showHostSettings(long(1), chatId, bot)
+                    showHostSettings(chatId, long(1), bot, long(0))
```

Comment:
In some places you use `int(1)` and `long(0)` in others
Both type of a parameter and its index swap